src/transforms/__snapshots__/block-folded.test.ts.snap

```diff
@@ -144,7 +144,7 @@ blockFolded (1:1 ~ 6:1)
   | ^
 6 | ¶
   | ^
-<blockFolded chomping="keep" indent=1 value="  123\\n  456\\n\\n\\n" />
+<blockFolded chomping="keep" indent=1 value=" 123\\n 456\\n\\n\\n" />
```

Comment:
The new value looks correct to me. What was the original value intended?

Reply:
I tried changing it to `String(blockValue.value)`, but it doesn't seem to make any difference to `blockValue.source` 🤔